tests/conftest.py

```diff
@@ -72,6 +72,7 @@
                   'tests.common.plugins.custom_fixtures',
                   'tests.common.dualtor',
                   'tests.decap',
+                  'tests.snmp',
```

Comment:
Why need to add this to the plugin? Contents defined in `tests.snmp` should be specific to the snmp test. We should only add common plugins that may be used by all tests. The `test_decap` is not a good example. We should refactor the code to remove `tests.decap` here as well.